config/config.go

```diff
@@ -260,14 +260,18 @@ func (c *Config) UnmarshalYAML(unmarshal func(interface{}) error) error {
 	// Do global overrides and validate unique names.
 	jobNames := map[string]struct{}{}
 	for _, scfg := range c.ScrapeConfigs {
+		if scfg.ScrapeTimeout > scfg.ScrapeInterval {
```

Comment:
I'm not sure about this. If I've set a global scrape interval (which is recommended), and then want to override the timeout I have to specify both in the job config.

I'd prefer that setting the scrape interval on a per-job level is avoided, as that tends to end in over-complication.